chat-plugins/scavengers.js

```diff
@@ -798,7 +798,7 @@ let commands = {
 			this.sendReply('|raw|' + str + "</table></div>");
 		} else {
 			const elapsed = Date.now() - game.startTime;
-			this.sendReplyBox(Chat.html`The current ${game.gameType ? `<em>${game.gameType}</em> ` : ""}scavenger hunt by <em>${Chat.toListString(game.hosts.map(h => h.name))}${(game.hosts.some(h => h.userid === game.staffHostId) ? '' : ` (started by - ${Chat.escapeHTML(game.staffHostName)})`)}</em> has been up for: ${Chat.toDurationString(elapsed, {hhmmss: true})}<br />Completed (${game.completed.length}): ${game.completed.map(u => Chat.escapeHTML(u.name)).join(', ')}<br />`);
+			this.sendReplyBox(Chat.html`The current ${game.gameType ? `<em>${game.gameType}</em> ` : ""}scavenger hunt by <em>${Chat.toListString(game.hosts.map(h => h.name))}${(game.hosts.some(h => h.userid === game.staffHostId) ? '' : ` (started by - ${game.staffHostName})`)}</em> has been up for: ${Chat.toDurationString(elapsed, {hhmmss: true})}<br />Completed (${game.completed.length}): ${game.completed.map(u => u.name).join(', ')}<br />`);
```

Comment:
The template string still hasn't been fixed...

Reply:
er what exactly is the problem here? I see travis is failing but its not giving me a reason why

Reply:
"You can't have <em> in template strings"

Reply:
The current version (before my commit) of this has <em> inside a template string, doesn't it?

Reply:
Explanation:

Here's an example of how to use ``Chat.html``:
```Javascript
const test = 'okay';
``Chat.html`````testing <em>${test}</em> `;``
```

Reply:
Specifically:

```js
const test = 'okay';
Chat.html`testing <em>${test}</em> `
```

generates the string `testing <em>okay</em>`, but

```js
const test = '<em>okay</em>';
Chat.html`testing ${test} `
```

generates `testing &lt;em&gt;okay&lt;/em&gt;`